getting-started/Installation/README.md

```diff
@@ -63,3 +64,6 @@ Optionally you can install as follow:
 brew tap amber-crystal/amber
 brew install amber-crystal/amber/amber
```

Comment:
This should now be:
```
brew install amberframework/amber/amber
```

@elorest - I don't know anything about creating homebrew packages. Is there any reason not to switch to:
```
brew install amberframework/amber
```